README.md

```diff
@@ -105,3 +111,17 @@ Here is the table for the support timeline:
 - `1.x.y`: released 2020-11-17, EOL
 - `2.x.y`: released 2022-02-10, supported until 2025-06-09
 - `3.x.y`: released 2023-06-09, current
+
+### Release notes
```

Comment:
I go back and forth on whether we should call them release notes. I feel like the open source community is used to calling them changelogs instead. What do you think?

Reply:
i'm not an expert but as a user i would at least i would always google for Release notes. Not sure. Had never heard "changelog" until i started contributing, though it was a long time ago.
If you think changelog is better, just submit the change, i'll commit it.

Reply:
Nah let's leave it as Release Notes then.  Thanks!